### PR TITLE
Integration Anistream

### PR DESCRIPTION
**Pull request by Flumuffel**

### CODE REVIEW

File: websites/A/Anistream/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Anistream",
+    "author": {
+      "name": "Flumuffel#3423",
+      "id": "274209471968837634"
+    },
+    "category": "anime",
+    "logo": "https://anistream.de/public/static/favicon.png",
+    "thumbnail": "https://anistream.de/public/static/Anistream.png",
```
Comment (EncryptedDev) [inline websites/A/Anistream/dist/metadata.json:10]: Logo and thumbnail must be on imgur

File: websites/A/Anistream/presence.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    clientId: "877640464151494676" //The client ID of the Application created at https://discordapp.com/developers/applications
+}),
+
+ browsingStamp = Math.floor(Date.now() / 1000);
+
+Anipresence.on("UpdateData", async () => {
+
+    /*UpdateData is always firing, and therefore should be used as your refresh cycle, or `tick`. This is called several times a second where possible.
+  
+      It is recommended to set up another function outside of this event function which will change variable values and do the heavy lifting if you call data from an API.*/
```
Comment (EncryptedDev) [inline websites/A/Anistream/presence.ts:11]: Remove these lines as they aren't needed

File: websites/A/Anistream/presence.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
+const Anipresence = new Presence({
+    clientId: "877640464151494676" //The client ID of the Application created at https://discordapp.com/developers/applications
+}),
+
```
Comment (EncryptedDev) [inline websites/A/Anistream/presence.ts:4]: Remove line

File: websites/A/Anistream/presence.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    };
+
+    if (document.location.hostname === "anistream.de") {
+        if (document.location.pathname === "/") {
+            presenceData.startTimestamp = browsingStamp;
+            presenceData.details = "Startseite";
+        } else if (document.location.pathname.includes("/serie/")) {
+
+            /*presenceData.startTimestamp = browsingStamp;*/
+
```
Comment (EncryptedDev) [inline websites/A/Anistream/presence.ts:24]: Remove these lines

File: websites/A/Anistream/presence.ts [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            const epNr = document.querySelector(".tab-pane.show.active > .active > .episode");
+            epName = document.querySelector(".tab-pane.show.active > .active > .name");
+
+
```
Comment (EncryptedDev) [inline websites/A/Anistream/presence.ts:28]: Remove these lines

File: websites/A/Anistream/presence.ts [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+            if (epName && epNr) {
+                presenceData.details = epName.innerHTML;
+                presenceData.state = epNr.innerHTML;
+            } else {
+                const title = document.querySelector("h1");
+                presenceData.details = title.innerText;
+                const staffel = document.querySelector(".nav-link.active");
+                presenceData.state = staffel.innerHTML;
+            }
+
```
Comment (EncryptedDev) [inline websites/A/Anistream/presence.ts:38]: Removed this

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Hi, remove all the comments on this code, it makes it messy and unreadable. Also, not much to this presence either

File: websites/A/Anistream/presence.ts (hunk 6 above)
Comment (Flumuffel) [reply websites/A/Anistream/presence.ts:38]: What should I remove here? The empty lines?

**Comment by Slowlife01** [comment]
It's very low quality (could be a lot better) atm so it won't be accepted anyway. So I would be better if you just rewrite a whole thing

**Pull request closed without merging**